### PR TITLE
Fixes #151 | Handle Mesos healthchecks

### LINKED ISSUE #151: Handle Mesos healthchecks
Mesos 1.1 introduces HTTP/S and TCP healthchecks. Marathon will support this healthchecks from 1.4. We need to support them too.

### PR DESCRIPTION
**Pull request by janisz**

### CODE REVIEW

File: consul/consul.go [hunk 1]
```diff
@@ -323,9 +323,13 @@ func (c *Consul) marathonToConsulChecks(task *apps.Task, healthChecks []apps.Hea
 		}
 
 		switch check.Protocol {
-		case "HTTP", "HTTPS":
+		case "HTTP", "HTTPS", "MESOS_HTTP", "MESOS_HTTPS":
```
Comment (adamdubiel) [inline consul/consul.go]: heh, what a name for a "protocol" (i guess it comes from Marathon)

File: consul/consul.go [hunk 2]
```diff
@@ -323,9 +323,13 @@ func (c *Consul) marathonToConsulChecks(task *apps.Task, healthChecks []apps.Hea
...
 			if parsedURL, err := url.ParseRequestURI(check.Path); err == nil {
-				parsedURL.Scheme = strings.ToLower(check.Protocol)
+				if check.Protocol == "HTTP" || check.Protocol == "MESOS_HTTP" {
```
Comment (adamdubiel) [inline consul/consul.go]: can you add a test for this?

Comment (janisz) [inline consul/consul.go]: What test would you like to see? I added one [here](https://github.com/allegro/marathon-consul/pull/157/files/0b2daba4b1ab7a3e0cf41131d52198b6e7a6795e#diff-eccf782d4b592a6a09965a14b71f5e08R832)

Comment (adamdubiel) [inline consul/consul.go]: oh, ok, diff mislead me. ack

**Pull request merged**